test/fixtures/sap.ui.core-buildtime/main/src/sap/ui/Global.js

```diff
@@ -0,0 +1,5 @@
+/*!
+ * ${copyright}
+ */
+// replacement of $\{buildtime\} is only made in sap/ui/Global.js
```

Comment:
```suggestion
// replacement of $\{buildtime\} is only made in sap/ui/Global.js and Global-dbg.js
```

The comment really confused me in the expected `Global-dbg.js`